File: angular/src/app/main/manufacturing/shared/dto/mftoolty.dto.ts
export class MFtooltySETDto {
    tenantId: number | null;
    tooltyid:string;
    id:number;
    tooltydesc: string;
    status: boolean;
    unit: string;
    unitcost: number;
    comments: string;
    audtUser: string;
    audtDate: Date | string | null;
    createdBy: string;
    createDate: Date | string | null;
}

export class GetMFtooltySETForViewDto {
    mfTooltyset: MFtooltySETDto
}

export class GetMFtooltySETDtoForEditOutput {
    mftoolty: MFtooltySETDto
}

export class PagedResultDtoMFtooltySET {
    totalCount: number;
    items: GetMFtooltySETForViewDto[]
}

export class CreateOrEditMFtooltySETDto {
    tenantId: number | null;
    tooltyid:string;
    tooltydesc: string;
    status: boolean;
    unit: string;
    unitcost: number;
    comments: string;
    audtUser: string;
    audtDate: Date | string | null;
    createDate: Date | string | null;
}

File: aspnet-core/src/ERP.Application.Shared/Manufacturing/SetupForms/Dtos/MFTOOLTYDto.cs
using System;
using Abp.Application.Services.Dto;

namespace ERP.Manufacturing.SetupForms.Dtos
{
    public class MFTOOLTYDto : EntityDto
    {
        public string TOOLTYID { get; set; }

        public string TOOLTYDESC { get; set; }

        public bool STATUS { get; set; }

        public double UNITCOST { get; set; }

        public string UNIT { get; set; }

        public string COMMENTS { get; set; }

        public string AudtUser { get; set; }

        public DateTime? AudtDate { get; set; }

        public string CreatedBy { get; set; }

        public DateTime? CreateDate { get; set; }

    }
}